components/ui/svg.tsx

```diff
@@ -50,6 +64,48 @@ export class SvgImage {
     return <Logo width={width} height={height} />;
   }
 
+  static kebabIcon() {
+    return <KebabIcon />;
+  }
+
+  static walletActivatedIcon() {
```

Comment:
Wallet Activated Icon and Wallet Un Activated Icon  seems similar, can we merge them both